pkg/runtime/interfaces.go

```diff
@@ -89,42 +89,36 @@ type Framer interface {
 
 // SerializerInfo contains information about a specific serialization format
 type SerializerInfo struct {
-	Serializer
-	// EncodesAsText indicates this serializer can be encoded to UTF-8 safely.
-	EncodesAsText bool
 	// MediaType is the value that represents this serializer over the wire.
 	MediaType string
+	// EncodesAsText indicates this serializer can be encoded to UTF-8 safely.
+	EncodesAsText bool
+	// Serializer is the individual object serializer for this media type.
+	Serializer
```

Comment:
I know this just moved from before, but would it be possible to change this to `Serializer Serializer`. Especially with this now encompassing 3 different serializers, is there a need for `SerializerInfo` to have `Serializer` methods on it?